src/SonicRangeSensor.cpp

```diff
@@ -8,11 +9,13 @@ int8_t SonicRangeSensor::setup(){
 
 //TODO: add long->byte[] casting in utils.h and use here!
 uint8_t SonicRangeSensor::read(uint8_t* address){
-  uint8_t len = 1;
-  long pulse_value, mm, inches;
+  uint8_t len;
+  long pulse_value, mm, inches,meow;
```

Comment:
I think we can get rid of most of these variables. `mm` is simply assigned to `pulse_value`, `inches` is not returned, and `meow` is never used after declaration. All we really need is `pulse_value`.